src/renderers/dom/shared/devtools/ReactDOMUnknownPropertyDevtool.js

```diff
@@ -83,32 +85,62 @@ if (__DEV__) {
         registrationName,
         ReactComponentTreeDevtool.getStackAddendumByID(debugID)
       );
+
+      return true;
     } else {
       // We were unable to guess which prop the user intended.
       // It is likely that the user was just blindly spreading/forwarding props
       // Components should be careful to only render valid props/attributes.
-      warning(
-        false,
-        'Unknown prop `%s` on <%s> tag. Remove this prop from the element. ' +
-        'For details, see https://fb.me/react-unknown-prop%s',
-        name,
-        tagName,
-        ReactComponentTreeDevtool.getStackAddendumByID(debugID)
-      );
+      // warning will be invoked in warnUnknownProperties to allow grouping
```

Comment:
Nit: previous two sentences started with capital and ended with period, let’s keep this one the same way?